client/qiskit_serverless/core/job.py

```diff
@@ -189,7 +189,10 @@ def filtered_logs(self, **kwargs) -> str:
 
     def error_message(self):
         """Returns the execution error message."""
-        return self._job_service.result(self.job_id) if self.status() == "ERROR" else ""
+        error = (
+            self._job_service.result(self.job_id) if self.status() == "ERROR" else ""
+        )
+        return error.strip('"').encode().decode("unicode_escape")
```

Comment:
I miss a `try/except` in case we could have problems transforming the string for whatever reason. I think we should always return the error as it is in case we cannot transforming it so the user can work with it.